cmd/snap/cmd_snap_op.go

```diff
@@ -69,7 +69,10 @@ func wait(client *client.Client, id string) (*client.Change, error) {
 			case t.ID == lastID:
 				pb.Set(float64(t.Progress.Done))
 			default:
-				pb.Start(t.Summary, float64(t.Progress.Total))
+				var pkgnames []string
+				chg.Get("snap-names", &pkgnames)
+				pkgname := pkgnames[0]
+				pb.Start(pkgname+" ", float64(t.Progress.Total))
```

Comment:
I wonder how this will look like in practice, I guess I just have to go and see...